client/src/main/java/io/spine/client/FilterValueConverter.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2018, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.client;
+
+import com.google.protobuf.Any;
+import io.spine.protobuf.TypeConverter;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * A utility for converting the {@link ColumnFilter} values represented as {@linkplain Object Java
+ * Objects} into the {@linkplain com.google.protobuf.Message Protobuf Messages} (in form of {@link
+ * Any}) and vice versa.
+ *
+ * <p>This class performs all necessary actions preceding the value conversion as well as the type
+ * conversion itself.
+ *
+ * <p>{@link ColumnFilter} values can either be {@linkplain com.google.protobuf.Message Protobuf
+ * Messages}, Java primitives, or {@link Enum enumerated values}.
+ *
+ * @author Dmytro Kuzmin
+ */
+public final class FilterValueConverter {
```

Comment:
See `Messages` and `TypeConverter` in `io.spine.protobuf` package (from `base` repository).

I think that most of the code in this class will go away. The rest should probably be non-public (i.e. package-private). Or some pieces should be moved to `base`. 

Let's discuss if needed.